masterlist.yaml

```diff
@@ -2336,15 +2336,20 @@ plugins:
     url: [ 'http://www.nexusmods.com/fallout4/mods/4772' ]
     after:
       - 'MoreSkinColors.esp'
+      - 'AzarPonyHairstyles.esp'
+      - 'Eli_Armour_Compendium.esp'
+      - 'Scrap Everything.+\.esp'
```

Comment:
After entries need to be exact plugin names, that regex (and the others in your commit) won't be used as a regex.